R/process_data_set_lizette.R

```diff
@@ -0,0 +1,14 @@
+library(tidyverse)
```

Comment:
This isn't the right file. You want to have the file with all of the documentation in the `R/` folder. Please see some of the recently accepted pull requests for examples (e.g., #65, #66, #67).

Reply:
You also need to add the data file in `data/` and update `DESCRIPTION` and `NEWS.md`, and rendered documentation files.